src/index.ts

```diff
@@ -192,6 +197,7 @@ export class ECRDeployment extends Construct {
         SrcCreds: props.src.creds,
         DestImage: props.dest.uri,
         DestCreds: props.dest.creds,
+        Architecture: props.architecture,
```

Comment:
I reckon we should fix this and move away from defaulting to the lambda architecture to explicitly defining a default here.